provider/resource.go

```diff
@@ -1880,10 +1800,12 @@ func Provider() tfbridge.ProviderInfo {
 
 			// Dashboard
 			// TODO: This resource is in the "Portal" module in the upstream provider. Move to a new namespace when terraform-provider-azurerm hits v3.0.
-			"azurerm_dashboard": {Tok: azureResource(azureDashboard, "Dashboard")},
+			"azurerm_dashboard":        {Tok: azureResource(azureDashboard, "Dashboard")},
```

Comment:
We should either move to the Portal module to be consistent with the upstream provider or remove the comment on the line above. If there's a reference for how we organize our modules, might be good to add to the README.

Reply:
Tell me what you want me to do here and we will do it :)

Reply:
Per Slack convo, the design intent is to mirror the upstream provider, so let's move to a Portal module.